Formula/r/ruby.rb

```diff
@@ -156,7 +156,7 @@ def post_install
     rm(%W[
       #{rubygems_bindir}/bundle
       #{rubygems_bindir}/bundler
-    ])
+    ].select { |file| File.exist?(file) })
     rm_r(Dir[HOMEBREW_PREFIX/"lib/ruby/gems/#{api_version}/gems/bundler-*"])
```

Comment:
does this line need protection too?

Reply:
I was just about to open a PR to fix this from https://github.com/Homebrew/homebrew-core/commit/46fc9416ff6a43da9cc32c986b102a44b9faae5f with the approach of rescuing from `Errno::ENOENT`

Reply:
`Dir["*"]` should return an empty array if it doesn't exist.

Reply:
👍